src/type-coercions.md

```diff
@@ -172,6 +175,9 @@ an implementation of `Unsize<U>` for `T` will be provided:
 
 * `T` to `dyn U`, when `T` implements `U + Sized`, and `U` is [object safe].
 
+* `dyn T` to `dyn U` when `U` is one of `T`'s [supertraits].
+    * This allows dropping auto traits, i.e. `dyn T + Auto` to `dyn U` is allowed.
```

Comment:
This also allows adding auto traits, i.e. `trait Sub: Sup + Auto` you can do `dyn Sub` -> `dyn Sup + Auto`. https://github.com/rust-lang/rust/pull/119338

Reply:
Added a mention of that